### DISCUSSION TITLE
installation guide using baidu paddlepaddle without tensorflow

### DISCUSSION THREAD

**Discussion by zzz2010** [deepmodeling/deepmd-kit#822] [Q&A]
Hello,
May I know if there is any installation guide in the environment using baidu paddlepaddle without tensorflow?

Thank you

**Comment by njzjz** [comment]
@JiabinYang @zhouwei25 Could you answer this question? Thank you.

**Reply by zzz2010** [reply]
Hi, especially, I want to know how to install deepmd-kit in Baidu AI studio which doesnt allow tensorflow installation.  thank you
